src/Studio/Plugins/Graph/GraphAttributesWidget.cpp

```diff
@@ -28,13 +28,36 @@
 #include <EngineManager.h>
 #include "../../AppManager.h"
 #include "../../MainWindow.h"
-
+#include <Engine/Core/AttributeSet.cpp>
```

Comment:
We will need to functionally test this @cyberjunk and make sure it doesn't introduce any regression bugs.

Reply:
@dioptre  To check the functionality how are you planning to test functionality without a device.